Patches/Mechanized Armor Set/Ammo_&_Apparel_Defs.xml

```diff
@@ -0,0 +1,139 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>Mechanized Armor Set</li>
+		</mods>
+
+		<match Class="PatchOperationSequence">
+			<operations>
+
+				
+				<!--Plate Helm-->
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/ThingDef[defName="Mechanized_Helm_Apparel"]/statBases</xpath>
+					<value>
+						<Bulk>100</Bulk>
+						<WornBulk>0</WornBulk>
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="Mechanized_Helm_Apparel"]/statBases/ArmorRating_Sharp</xpath>
+					<value>
+						<ArmorRating_Sharp>3.5</ArmorRating_Sharp>
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="Mechanized_Helm_Apparel"]/statBases/ArmorRating_Blunt</xpath>
+					<value>
+						<ArmorRating_Blunt>5.2</ArmorRating_Blunt>
+					</value>
+				</li>
+
+				
+				<!--Plate Armour-->
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/ThingDef[defName="Mechanized_Plate_Apparel"]/statBases</xpath>
+					<value>
+						<Bulk>100</Bulk>
+						<WornBulk>0</WornBulk>
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="Mechanized_Plate_Apparel"]/statBases/ArmorRating_Sharp</xpath>
+					<value>
+						<ArmorRating_Sharp>7.2</ArmorRating_Sharp>
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="Mechanized_Plate_Apparel"]/statBases/ArmorRating_Blunt</xpath>
+					<value>
+						<ArmorRating_Blunt>5.5</ArmorRating_Blunt>
+					</value>
+				</li>
+				

```

Comment:
Go ahead and add these two operations to bring it in line with other CE "power" armor. (And confirm that they work, please.)

```
	<li Class="PatchOperationAdd">
		<xpath>Defs/ThingDef[defName="Mechanized_Plate_Apparel"]/apparel/bodyPartGroups</xpath>
		<value>
			<li>Hands</li>
			<li>Feet</li>
		</value>
	</li>

	<li Class="PatchOperationAddModExtension">
		<xpath>Defs/ThingDef[defName="Mechanized_Plate_Apparel"]</xpath>
		<value>
		  <li Class="CombatExtended.PartialArmorExt">
			  <stats>
				  <li>
					<ArmorRating_Sharp>0.60</ArmorRating_Sharp>
					<parts>
						<li>Neck</li>
					</parts>
				  </li>
				  <li>
					<ArmorRating_Blunt>0.60</ArmorRating_Blunt>
					<parts>
						<li>Neck</li>
					</parts>
				  </li>
				  <li>
					<ArmorRating_Sharp>0.90</ArmorRating_Sharp>
					<parts>
						<li>Leg</li>
					</parts>
				  </li>
				  <li>
					<ArmorRating_Blunt>0.90</ArmorRating_Blunt>
					<parts>
						<li>Leg</li>
					</parts>
				  </li>
				  <li>
					<ArmorRating_Sharp>0.80</ArmorRating_Sharp>
					<parts>
						<li>Arm</li>
					</parts>
				  </li>
				  <li>
					<ArmorRating_Blunt>0.80</ArmorRating_Blunt>
					<parts>
						<li>Arm</li>
					</parts>
				  </li>
				  <li>
					<ArmorRating_Sharp>0.60</ArmorRating_Sharp>
					<parts>
						<li>Hand</li>
					</parts>
				  </li>
				  <li>
					<ArmorRating_Blunt>0.60</ArmorRating_Blunt>
					<parts>
						<li>Hand</li>
					</parts>
				  </li>
			  </stats>
		  </li>
		</value>
	</li>
```